scripts/testone.sh

```diff
@@ -27,7 +27,6 @@ if [[ "$#" -eq "1" ]]; then
 else 
     TEST_CASE="$TEST_FILE:$2"
 fi
-ulimit -s 65532 && (ulimit -n 10240 || true) && \
```

Comment:
I don't know whether this change should be committed in the end. But this line seems to *need* to be deactivated on MacOS.

@mrmr1993 Do you know why we'd actually need to *always* increase the stack size and file descriptors when invoking `dune` -- AFAIK, it's also everywhere in the Makefile but I never ran into issues without having this set up.

Reply:
Oh thanks for pointing out, you're right that I am only making this change for my local tests, but will undo it before merging

Reply:
Some devs (and CI, at some point IIRC) used to have issues with this. I don't know if that's true, but I'd be inclined to leave it, since it's not harming anyone.

I *do* know that we've had to explicitly increase the stack size for js_of_ocaml recently, so it may still be actively helping elsewhere too

Reply:
Oh this works for me

Reply:
Better than a OS version inspection: `(ulimit -s 65532 || true)`.

Reply:
Even better. yeah